docs/reference/filebeat/filebeat-input-journald.md

```diff
@@ -110,8 +112,24 @@ input will only ingest the journal files found when started. New
 files will not be ingested.
 :::
 
+### `chroot` [filebeat-input-journald-chroot]
+```{applies_to}
+stack: ga 9.3.0
+```
+A folder to be used as chroot when calling `journalctl`. This allows
+Filebeat to call the host's `journalctl` directly. If using this
+option, {{filebeat}} must be run as root and
```

Comment:
The wolfi containers are not root by default, you will need examples of how to give the capabilities needed to do this. 

You potentially only need `CAP_SYS_CHROOT` and not all the root capabilities, we should identify what the least privileged way for this to execute is.